### ISSUE TITLE
Undefined method 'level' for GroceryDelivery::Log::Module

### ISSUE THREAD

**Issue by johnnyramos** [facebook/grocery-delivery#54]
It might be something I'm doing wrong in my latest setup but I'm getting this repeated error on the latest version of `chefdk` and `chef-workstation`.

OS Version: `Ubuntu 16.04.6 LTS`
grocery_delivery Gem: `0.0.8`

Chef-DK Version: `4.8.38`
```
WARN: Dryrun mode activated, no changes will be made.
WARN: Attempting to acquire lock
WARN: Lock acquired
WARN: [DRYRUN] Would do: Updating repo
ERROR: /opt/chefdk/embedded/lib/ruby/gems/2.6.0/gems/between_meals-0.0.11/lib/between_meals/knife.rb:44:in `initialize'
ERROR: /opt/chefdk/embedded/lib/ruby/gems/2.6.0/gems/grocery_delivery-0.0.8/bin/grocery-delivery:151:in `new'
ERROR: /opt/chefdk/embedded/lib/ruby/gems/2.6.0/gems/grocery_delivery-0.0.8/bin/grocery-delivery:151:in `upload_changed'
ERROR: /opt/chefdk/embedded/lib/ruby/gems/2.6.0/gems/grocery_delivery-0.0.8/bin/grocery-delivery:281:in `'
ERROR: /opt/chefdk/embedded/bin/grocery-delivery:23:in `load'
ERROR: /opt/chefdk/embedded/bin/grocery-delivery:23:in `'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `load'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `kernel_load'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/cli/exec.rb:28:in `run'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/cli.rb:463:in `exec'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/cli.rb:27:in `dispatch'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/cli.rb:18:in `start'
ERROR: /opt/chefdk/embedded/lib/ruby/gems/2.6.0/gems/bundler-1.17.2/exe/bundle:30:in `block in '
ERROR: /opt/chefdk/embedded/lib/ruby/2.6.0/bundler/friendly_errors.rb:124:in `with_friendly_errors'
ERROR: /opt/chefdk/embedded/lib/ruby/gems/2.6.0/gems/bundler-1.17.2/exe/bundle:22:in `'
ERROR: /opt/chefdk/embedded/bin/bundle:23:in `load'
ERROR: /opt/chefdk/embedded/bin/bundle:23:in `'
WARN: undefined method `level' for GroceryDelivery::Log:Module
```

Chef Workstation Version: `20.6.62`
```
WARN: Dryrun mode activated, no changes will be made.
WARN: Attempting to acquire lock
WARN: Lock acquired
WARN: [DRYRUN] Would do: Updating repo
ERROR: /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/between_meals-0.0.11/lib/between_meals/knife.rb:44:in `initialize'
ERROR: /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/grocery_delivery-0.0.8/bin/grocery-delivery:151:in `new'
ERROR: /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/grocery_delivery-0.0.8/bin/grocery-delivery:151:in `upload_changed'
ERROR: /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/grocery_delivery-0.0.8/bin/grocery-delivery:281:in `'
ERROR: /opt/chef-workstation/embedded/bin/grocery-delivery:23:in `load'
ERROR: /opt/chef-workstation/embedded/bin/grocery-delivery:23:in `'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `load'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/cli/exec.rb:63:in `kernel_load'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/cli/exec.rb:28:in `run'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/cli.rb:476:in `exec'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/cli.rb:30:in `dispatch'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/cli.rb:24:in `start'
ERROR: /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/libexec/bundle:46:in `block in '
ERROR: /opt/chef-workstation/embedded/lib/ruby/2.7.0/bundler/friendly_errors.rb:123:in `with_friendly_errors'
ERROR: /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/libexec/bundle:34:in `'
ERROR: /opt/chef-workstation/embedded/bin/bundle:23:in `load'
ERROR: /opt/chef-workstation/embedded/bin/bundle:23:in `'
WARN: undefined method `level' for GroceryDelivery::Log:Module
```

**Comment by NaomiReeves** [comment]
I believe this was fixed in #51 but there hasn't been a `grocery-delivery` release since that was merged. I'll cut a new release today.

**Comment by NaomiReeves** [comment]
I've released a new gem, [v0.0.9](https://github.com/facebook/grocery-delivery/releases/tag/v0.0.9). Let me know if this fixes the error.

**Comment by johnnyramos** [comment]
We are good now, thanks for the quick turn around! :D
